metadata.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "puppetlabs-ruby_task_helper",
+  "version": "0.1.0",
+  "author": "lucy",
+  "summary": "",
```

Comment:
Summary and source need to be filled in (as I just found out). Otherwise the forge will reject it.